src/activities/quizzes/QuizEntity.js

```diff
@@ -1249,7 +1290,8 @@ export class QuizEntity extends Entity {
 				return false;
 			}
 		}
-		return true;
+
+		return isEqual(this.remediationCandidates(), quiz.remediationCandidates);
```

Comment:
Does this work when `isStudySupportEnabledVisible` is `false`?
I'm wondering if the comparison will be: `isEqual(undefined, null);`

Reply:
wouldn't both be undefined though? Because quiz.remediationCandidates is initialized to entity.remediationCandidates(), so they should be the same?